src/client/components/card/components/Card.jsx

```diff
@@ -1,14 +1,86 @@
+import React, { PropTypes } from 'react'
 import { Card } from 'material-ui/Card'
 import styled from 'styled-components'
 
+const SIZE = { small: 'small', medium: 'medium', large: 'large' }
+
+// Determine the width based on either a fixed width or a relative size indicator.
+const determineWidth = ({ width, size }) => {
+  let cssWidth = '100%'
+  if (width) {
+    cssWidth = width
+  } else if (size) {
+    // Note: these should change based on the viewport (should be responsive)
+    switch (size) {
+      case SIZE.small:
+        cssWidth = '40rem'
+        break
+      case SIZE.medium:
+        cssWidth = '60rem'
+        break
+      case SIZE.large:
+        cssWidth = '90rem'
+        break
+      default:
+        throw new Error(`Unknown size ${size}`)
+    }
+  }
+  return cssWidth
+}
+
 const CardWrapper = styled(Card)`
   margin: ${props => props.marginTop} auto 0;
-  width: ${props => props.width};
+  padding: ${props => props.padding};
+  width: ${props => determineWidth(props)};
 `
 
+CardWrapper.propTypes = {
+  title: PropTypes.string,
+  marginTop: PropTypes.string,
+  padding: PropTypes.string,
+  width: PropTypes.string,
+  size: PropTypes.oneOf(Object.values(SIZE))
+}
+
 CardWrapper.defaultProps = {
-  width: '25rem',
-  marginTop: '5rem'
+  marginTop: '5rem',
+  padding: '2rem'
+}
+
+const CardTitle = styled.div`
+  background: #2c3e50;
+  color: #fff;
+  font-size: 1.6rem;
+  padding: 1rem;
+  text-align: center;
+`
+
+const CardWithTitle = props => {
```

Comment:
Shouldn't this just be Card? Since it can also not have a title? I assume that this is because of the Card from material-ui, but in that case maybe rename the material-ui import to distinguish between their and our Card.